packages/client/src/index.js

```diff
@@ -233,7 +227,7 @@ export let ContextTree = _.curry(
           if (debug && node._meta) target.metaHistory.push(node._meta)
         }
 
-        target.hasResults = hasResults(snapshot)(target)
+        target.hasResults = !!Tree.findNode(F.isNotBlank, target.context)
```

Comment:
I would still prefer it to be in a separate function. In any case you still need to `snapshot(target.context)`

Reply:
Done